runtime/tests/integration_tests.rs

```diff
@@ -191,12 +191,37 @@ fn test_fips_cmd_api() {
     let mut model = run_rom_test("mbox");
     let expected_err = Err(ModelError::MailboxCmdFailed(0x000E0006));
 
+    model
+        .soc_ifc()
+        .cptra_fw_rev_id()
+        .at(0)
+        .write(|_| b'R'.into());
+
+    model
+        .soc_ifc()
+        .cptra_fw_rev_id()
+        .at(1)
+        .write(|_| b'A'.into());
+
     model.step_until(|m| m.soc_mbox().status().read().mbox_fsm_ps().mbox_idle());
 
     let cmd = [0u8; 4];
 
-    let resp = model.mailbox_execute(u32::from(CommandId::VERSION), &cmd);
-    assert_eq!(resp, expected_err);
+    let fips_version_resp = model
+        .mailbox_execute(u32::from(CommandId::VERSION), &cmd)
+        .unwrap()
+        .unwrap();
+
+    pub const NAME: &str = "Caliptra RTM";
+
+    // Check command size
+    let fips_version: &[u8] = fips_version_resp.as_bytes();
+    assert_eq!(fips_version.len(), 28);
+
+    // Check module name
+    let name = &fips_version[16..28];
+    let result = name.cmp(NAME.as_bytes());
+    assert!(result.is_eq());
```

Comment:
Might be easier to do `let fips_version = VersionResponse::read_from(fips_version_resp.as_bytes());`. Then you can read individual fields.

Reply:
Done!